src/Todo.tsx

```diff
@@ -1,10 +1,42 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
+import CreateList from './CreateList';
+
+export async function GetTodoList() {
+    const response = await fetch('http://127.0.0.1:8000/todo-list/')
```

Comment:
Do you have anything in the tests stubbing out HTTP requests? It looks like this is actually making a request to the server, and it shouldn't.

Reply:
Should be fixed now